src/makeCalculator.js

```diff
@@ -36,8 +36,41 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
-}
+  const calculator = {
+    reset: function() {
+      calculator.result = 0;
+
+      return this;
+    },
+
+    add: function(n) {
+      calculator.result += n;
+    },
+
+    subtract: function(n) {
```

Comment:
```suggestion
    add(n) {
      calculator.result += n;
    },

    subtract(n) {
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/219701067-bf6e6a74-275d-4d9c-80aa-d6874af29ecd.png)